CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
 
+
+## [0.2.2] - 2022-09-28
```

Comment:
Please add the following to the bottom of this file otherwise the hyperlink in the changelog won't work (see current [changelog](https://github.com/datajoint/element-calcium-imaging/blob/main/CHANGELOG.md) version `0.2.0` versus `0.2.1`).

```
[0.2.2]: https://github.com/datajoint/element-calcium-imaging/releases/tag/0.2.2
[0.2.1]: https://github.com/datajoint/element-calcium-imaging/releases/tag/0.2.1
```

Reply:
Added them.

> [0.2.2]: https://github.com/datajoint/element-calcium-imaging/releases/tag/0.2.2
> [0.2.1]: https://github.com/datajoint/element-calcium-imaging/releases/tag/0.2.1